mobilepopup.css

```diff
@@ -0,0 +1,69 @@
+@media screen and (max-width: 420px) {
+  .surname {
+    display: none;
+  }
+
+  .cardarticle {
+    color: rgba(103, 121, 142, 1);
+  }
+
+  .button-section {
+    display: flex;
+    column-gap: 24px;
+    width: fit-content;
+    align-items: center;
+    list-style: none;
+    padding-top: 16px;
+  }
+
+  .cardlinks {
+    display: flex;
+    flex-direction: row;
+    list-style: none;
+    padding: 0;
+    box-sizing: border-box;
+    margin-bottom: 12px;
+    gap: 12px;
+    text-align: center;
+  }
+
+  ul .linkBtn {
+    align-items: center;
+    height: 36px;
+    border: 1px solid #979493;
+    text-align: center;
+    justify-content: center;
+    display: flex;
+    padding: 0 10px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    color: hsl(20, 3%, 23%);
+  }
```

Comment:
- [ ] For the line below the mobile menu options, please increase the width of it to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://github.com/richie1988/Microverse-websitepagesproject/assets/68796232/25def5e6-65e7-4c26-a79d-f62c78e2a68f) | ![image](https://github.com/richie1988/Microverse-websitepagesproject/assets/68796232/02c96ab3-c7a9-491f-99e0-64b91274688c)  |